website/search/search.py

```diff
@@ -51,6 +51,10 @@ def update_user(user, index=None):
     index = index or settings.ELASTIC_INDEX
     search_engine.update_user(user, index=index)
 
+@requires_search
+def update_file(file_, index=None, delete=False):
+    index = index or settings.ELASTIC_INDEX
```

Comment:
@abought We have used this pattern so that the setting can be modified at runtime, e.g. in tests.

```python
settings.ELASTIC_INDEX = 'mycoolindex'
update_file(file)
```